plugin-management-cli/pom.xml

```diff
@@ -39,6 +39,12 @@
             <version>1.0.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.16.1</version>
```

Comment:
would be good to manage this version in the parent in a `dependencyManagement` block

as it's shared between the cli and library

Reply:
Done. I updated the commit.